tests/publiccloud/run_ltp.pm

```diff
@@ -0,0 +1,147 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Use perl script to run LTP on public cloud
+#
+# Maintainer: Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base "publiccloud::basetest";
+use strict;
+use testapi;
+use utils;
+use serial_terminal 'select_virtio_console';
+use Data::Dumper;
+
+sub assert_ssh
+{
+    my ($instance, $cmd) = @_;
+
+    my $ssh_cmd = 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ';
+    $ssh_cmd .= '-i ' . $instance->{ssh_key} . ' ';
+    $ssh_cmd .= $instance->{username} . '@' . $instance->{ip} . ' ';
+    $ssh_cmd .= '-- ' . $cmd;
+
+    assert_script_run($ssh_cmd);
```

Comment:
maybe I'm wrong, but should we return the value that the command returns? Something like:
    return assert_script_run($ssh_cmd);

but maybe it's ok as it is.

Reply:
afaik assert_script_run() doesn't return, as the test quit if it fails

Reply:
it is assert_script_run no point to return something , if it fails it fails whole test/module